velox/vector/VectorStream.h

```diff
@@ -75,6 +75,8 @@ bool registerVectorSerde(std::unique_ptr<VectorSerde> serde);
 
 bool isRegisteredVectorSerde();
 
+std::unique_ptr<VectorSerde>& getVectorSerde();
```

Comment:
Would it make sense to return a raw pointer here?

Reply:
+1